.github/actions/create_manifest/action.yml

```diff
@@ -0,0 +1,43 @@
+name: 'Create manifest'
+description: 'Creates manifest containing versions of the product and the corresponding repositories'
+inputs:
+  repos:
+    description: "Multi-line list of repositories to include to manifest"
+    required: true
+  product_type:
+    description: "Unique string to reflect product configuration"
+    required: true
+  save_to:
+    description: "Path to save manifest to"
+    required: true
+  action_path:
```

Comment:
why do we need to set action path? when it could be changed?

Reply:
Due to issues with workspace paths on Azure runners (same one because of which we set GITHUB_WORKSPACE in our regular jobs)